dataflow/src/main/java/org/checkerframework/dataflow/cfg/node/CaseNode.java

```diff
@@ -2,9 +2,8 @@
 
 import com.sun.source.tree.CaseTree;
 import com.sun.source.tree.Tree.Kind;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.Objects;
+import java.util.*;
```

Comment:
Our policy is not to use wildcard imports.  Love it or hate it, could you please revert this change?